src/dfs/ipfs.spec.ts

```diff
@@ -110,6 +111,22 @@ describe('IPFS handler', function test() {
     delete ipfs.cache;
   });
 
+  it('should not be able to add a file when misconfigured', async () => {
+    ipfs.remoteNode = new IpfsLib({ host: 'ipfs.test.evan.network', port: '600', protocol: 'https' });
+    const randomContent = Math.random().toString();
+    const hash = ipfs.add('test', Buffer.from(randomContent, 'utf-8'));
+    await expect(hash).to.be.rejected;
```

Comment:
Would it be better to explicitly check for the error message too? Cause you never know with our tests... ;)
So
`expect.to.be.rejectedWith(...)`